src/index.ts

```diff
@@ -2439,6 +2441,8 @@ export interface GetConsentRequest {
 
   collectedAt?: number
   isGpcEnabled?: boolean
+  purposeLegitimateInterests?: { [key: string]: string }
```

Comment:
Can we make the value of the string an enum? similar to how VendorConsent is `export type VendorConsent = { [key: string]: VendorStatus }`